docs/internals/contributing/triaging-tickets.txt

```diff
@@ -296,7 +296,7 @@ If you do close a ticket, you should always make sure of the following:
 A ticket can be resolved in a number of ways:
 
 * fixed
-      Used once a patch has been rolled into Django and the issue is fixed.
+      Used once a pull request has been merged and the issue is fixed.
```

Comment:
```suggestion
      Used once a contribution has been merged and the issue is fixed.
```
Maybe even "commit" :thinking:

Reply:
I went for commit, it seemed (marginally) better to me 👍🏻